src/include/storage/write_ahead_log/log_manager.h

```diff
@@ -7,11 +7,15 @@
 #include <vector>
 #include "common/spin_latch.h"
 #include "common/strong_typedef.h"
+#include "spdlog/details/mpmc_blocking_q.h"
```

Comment:
The use of this seems like a bad idea. Debug logging is supposed to be for debug logging only.

If you want to use this data structure, write a replaceable wrapper (see what we did for concurrent hash table and spin latch)